src/frontend/screens/login/Login.jsx

```diff
@@ -1,7 +1,191 @@
-import React from "react";
+import React, { useState } from "react";
+import { Link, useNavigate, useLocation } from "react-router-dom";
+import loginImg from "../../assets/loginImg.svg";
+import { useDispatch } from "react-redux";
+import { token } from "../../actions";
+import axios from "axios";
+import { toast } from "react-toastify";
 
 function Login() {
-	return <div>Login page</div>;
+	const [userDetail, setUserDetail] = useState({
+		email: "",
+		password: "",
+	});
+	const dispatch = useDispatch();
+	const navigate = useNavigate();
+	const location = useLocation();
+	const handleChange = (e) => {
+		setUserDetail({
+			...userDetail,
+			[e.target.name]: e.target.value,
+		});
+	};
+	let from = location.state?.from?.pathname || "/";
+	const loginHandler = async (email, password, dispatch) => {
+		try {
+			const response = await axios.post(`/api/auth/login`, {
+				email,
+				password,
+			});
+
+			localStorage.setItem("token", response.data.encodedToken);
+			dispatch(
+				token({
+					token: response.data.encodedToken,
+					userInfo: response.data.foundUser,
+				})
+			);
+			toast.success("Logged in Successfully ", {
+				position: "top-right",
```

Comment:
This styling can be abstracted out in a separate file.

Reply:
You can have a styles.js and keep adding your styles there.

```
export const Tracker = {
    toastStyle : {
        position: "top-right",
       autoClose: 2000,
    },
....
}
```

And then pass the object here, 
toast.error("Wrong Email", {...Tracker.toastStyle});

Reply:
yes added this , instead of repeating  code , thanks for pointing  this out

Reply:
I think you can pass the overall style in toastcontainer. No need to pass the sty;es in each toast.